### ISSUE TITLE
Not possible to search an array of strings (tags)

### ISSUE THREAD

**Issue by kristof** [ForestAdmin/forest-express-mongoose#24]
I'm having trouble to search through an array of strings, defined in my model as shown below.

```javascript
var Product = new Schema({
  title: { type: String, required: true },
  description: { type: String, required: true },
  tags: [{type: String}],
  createdAt: { type: Date },
  updatedAt: { type: Date }
});
```

The field does appear within Forest admin but as a comma separated text field, however it is not possible to create segments or search on this field.

**Comment by arnaudbesnier** [comment]
Hey @kristof,

As you notice, the search/filters on array of String is not something we support yet.
We will definitely think about it for the future.

**Comment by kristof** [comment]
@arnaudbesnier thanks for the quick response! Is there a way to return custom result sets, so that I can build the query myself and return the results to Forest admin?

**Comment by arnaudbesnier** [comment]
Hey @kristof, we will start the implementation of the "Smart Segment" feature before the end of the week. It should give you all the flexibility to create the custom set of results you need.

🎄🎄🎄

**Comment by kristof** [comment]
@arnaudbesnier You sir, made my day! Looking forward to it.

**Comment by arnaudbesnier** [comment]
This feature has been released in the package version 1.0.4 !

🌲🌲🌲
